botstrap.py

```diff
@@ -33,85 +40,138 @@
     raise ValueError(message)
 
 
+class SilencedDict(dict):
+    """A dictionary that silences KeyError exceptions upon subscription to non existent items."""
+
+    def __init__(self, name: str):
+        self.name = name
+        super().__init__()
+
+    def __getitem__(self, item: str):
+        try:
+            super().__getitem__(item)
+        except KeyError:
+            log.warning(f"Couldn't find key: {item} in dict: {self.name} ")
+            log.warning("Please make sure to use our template: https://discord.new/zmHtscpYN9E3"
+                        "to make your own copy of the server to guarantee a successful run of botstrap ")
```

Comment:
I think I would rather point to the contributing guide than to hardcode the template, but it's not a big deal.

Reply:
Good point, i will use that.

Reply:
dace921097e56dc39d6643f384d1ca08befc9560